tests/cfgs/default/result/electronicarts.pcapng.out

```diff
@@ -0,0 +1,34 @@
+DPI Packets (TCP):	16	(8.00 pkts/flow)
```

Comment:
Instead of adding a new trace (with trivial TLS traffic), could you add these two flows to the existing `sites.pcap` file, please?